features/inventory/viewing_product_variant_with_specify_quantity_of_items_on_hand.feature

```diff
@@ -0,0 +1,43 @@
+@inventory
+Feature: Seeing product's variant with specify quantity of items on hand
+    In order to see how many variants of selected product are available on hand
+    As an Administrator
+    I want to be able to see quantity of variants available on hand for a selected product
```

Comment:
`I want to be able to see the on hand quantity of a selected/given product variant`